lib/src/tar/export.rs

```diff
@@ -26,6 +26,8 @@ pub const FORMAT_VERSIONS: RangeInclusive<u32> = 0..=1;
 const SYSROOT: &str = "sysroot";
 // This way the default ostree -> sysroot/ostree symlink works.
 const OSTREEDIR: &str = "sysroot/ostree";
+// The ref added (under ostree/) in the exported OSTree repo pointing at the commit.
+const OSTREEREF: &str = "encapsulated";
```

Comment:
I think this should be `ostree/encapsulated` though; that's what the ref is.  It's the string you'd pass to `ostree rev-parse`.